website/content/en/highlights/2023-11-07-new-linux-repos.md

```diff
@@ -0,0 +1,124 @@
+---
+date: "2023-11-07"
+title: "A New Home for Linux Packages"
+description: ""
+authors: ["spencergilbert"]
+pr_numbers: []
+release: "0.34.0"
+hide_on_release_notes: false
+badges:
+  type: "announcement"
+---
+
+Starting with Vector's `0.34.0` release, the `deb` and `rpm` packages will have
+a new home at `vector.dev` courtesy of Datadog. We'll continue to publish
+packages to both the new and existing locations through the end of **2023**.
+
+{{< warning >}}
+Starting in **2024**, we will stop using `timber.io` and exclusively publish packages
+to `vector.dev`.
+{{< /warning >}}
+
+We aim to make this a seamless transition by providing plenty of time for you to switch
+repositories, as well as by publishing previous versions of Vector with the new
+repository as a drop-in replacement, so you don't have to update your Vector version.
+
+If you have any questions or concerns don't hesitate to reach out on [Discord]
+or [Discussions]!
+
+## Migration guide
+
+The following command **removes** the existing repository and configures the
+new repository.
+
+```sh
+CSM_MIGRATE=true bash -c "$(curl -L https://s3.amazonaws.com/dd-agent/scripts/install_script_vector0.sh)"
```

Comment:
When running this script for the RPM installation, it prompted me when running `yum install vector` to accept the new keys. Is that expected? I somewhat expected this script to trust the keys already.